index.bs

```diff
@@ -1076,8 +1078,10 @@ The <dfn method for=FileSystemSyncAccessHandle>read(|buffer|, {{FileSystemReadWr
 1. Let |bufferSize| be |buffer|'s [=byte length=].
 1. Let |fileContents| be [=this=].[=FileSystemSyncAccessHandle/[[file]]=]'s [=file entry/binary data=].
 1. Let |fileSize| be |fileContents|'s [=byte sequence/length=].
-1. Let |readStart| be |options|.{{FileSystemReadWriteOptions/at}}.
-1. If |readStart| is larger than |fileSize|, return 0.
+1. Let |readStart| be |options|.{{FileSystemReadWriteOptions/at}}, if given; otherwise [=this=]'s [= FileSystemWritableFileStream/file position cursor=].
```

Comment:
```suggestion
1. Let |readStart| be |options|["{{FileSystemReadWriteOptions/at}}"], if given; otherwise [=this=]'s [=FileSystemWritableFileStream/file position cursor=].
```